draft-ietf-tls-dtls-connection-id.md

```diff
@@ -162,158 +161,64 @@ connection identifiers to other parties.
-  } DTLSInnerPlaintext;
-
-  struct {
-     ContentType opaque_type = 23; /* application_data */
-     ProtocolVersion legacy_record_version = {254,253); // DTLSv1.2
-     uint16 epoch;                         // DTLS-related field
-     uint48 sequence_number;               // DTLS-related field
-     opaque cid[cid_length];               // New field
-     uint16 length;
-     opaque encrypted_record[length];
-  } DTLSCiphertext;
-~~~~
-{: #dtls-record13 title="DTLS 1.3 Record Format with Connection ID"}
-
-Besides the "cid" field, all other fields are defined in the DTLS 1.2 and
-DTLS 1.3 specifications.
-
 Note that for both record formats, it is not possible to parse the
-records without knowing if the connection ID is in use and how long
-it is.
-
-# Examples
+records without knowing how long the Connection ID is.
 
-Below is an example exchange for DTLS 1.3 using a single
-connection id in each direction.
+In order to help the receiver discerning a record bearing a Connection ID from
+one that doesn't, the ContentType of the record containing the cid field MUST
+be: application_data_with_cid(25) instead of application_data(23),
+alert_with_cid(26) instead of alert(24) and heartbeat_with_cid(27) instead of
+heartbeat(24).
 
-~~~~
-Client                                             Server
-------                                             ------
-
-ClientHello
-(connection_id=5)
-                            -------->
-
-
-                            <--------       HelloRetryRequest
-                                                     (cookie)
-
-ClientHello                 -------->
-(connection_id=5)
-  +cookie
-
-                            <--------             ServerHello
-                                          (connection_id=100)
-                                          EncryptedExtensions
-                                                      (cid=5)
-                                                  Certificate
-                                                      (cid=5)
-                                            CertificateVerify
-                                                      (cid=5)
-                                                     Finished
-                                                      (cid=5)
-
-Certificate                -------->
-(cid=100)
-CertificateVerify
-(cid=100)
-Finished
-(cid=100)
-                           <--------                      Ack
-                                                      (cid=5)
-
-Application Data           ========>
-(cid=100)
-                           <========         Application Data
-                                                      (cid=5)
-~~~~
-{: #dtls-example title="Example DTLS 1.3 Exchange with Connection IDs"}
+# Examples
 
-Below is an example exchange for DTLS 1.2 using a connection id used
-uni-directionally from the client to the server.
+{{dtls-example2}} shows an example exchange where a connection id used
```

Comment:
I don't see this example.

Reply:
It's the text just below the text.  In the rendered TXT it's "Figure 2".

Reply:
OK.